packages/core/scripts/hardhat/tasks/registry.js

```diff
@@ -0,0 +1,33 @@
+const { task } = require("hardhat/config");
+const { RegistryRolesEnum } = require("@uma/common");
+const { waitForTxn } = require("./utils");
+
+task("register-deployer", "Register deployer with Registry capable of making price requests").setAction(async function(
```

Comment:
Do these automatically get registered as tasks because they're in the tasks directory?

Reply:
because theyre imported in `hardhat.config.js`. Note the `require("/path/to/registry.js")` at the bottom

Reply:
Makes sense. I think there are some disadvantages to this over hardhat scripts, but we can talk more about that later.